lab4-new.java/src/StringSearch.java

```diff
@@ -17,7 +17,7 @@ public static ArrayList<String> search(String word, File file) {
 
             while (scanner.hasNextLine()) {
```

Comment:
You do not handle error reading in file

Reply:
We think that we do handle this with a try-catch, but we have changed to print an error message also.